src/test/java/specs/TimeoutSpecs.java

```diff
@@ -0,0 +1,92 @@
+package specs;
+
+import static com.greghaskins.spectrum.Configure.timeout;
+import static com.greghaskins.spectrum.Configure.with;
+import static com.greghaskins.spectrum.Spectrum.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.greghaskins.spectrum.Spectrum;
+import com.greghaskins.spectrum.SpectrumHelper;
+
+import org.junit.runner.Result;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.TimeUnit;
+
+@RunWith(Spectrum.class)
+public class TimeoutSpecs {
+  {
+    describe("A suite with timeouts", () -> {
+      it("will allow things to pass if they run quicker than the timeout", () -> {
+        final Result result = SpectrumHelper.run(() -> {
```

Comment:
Using this approach rather than method-level classes, as I think it reads better... feedback appreciated.

Reply:
Agreed. I added that test helper for this exact purpose, as I got sick of writing nested classes.

Reply:
The nested classes were a rather surprising bit of java syntax - cleverly found!